mentors/python/g.json

```
@@ -1,10 +1,18 @@
 [
-{
-    "github_username": "gccarcadia",
-    "name": "George-Claudiu Carcadia",
-    "link_text": null,
-    "link_url": null,
-    "avatar_url": null,
-    "bio": "I've been using Python since two and a half years ago and I really fell in love with it. I really hope that I can share my knowledge with others and also become more knowledgeable myself along the way." 
-}
+    {
+        "github_username": "gccarcadia",
+        "name": "George-Claudiu Carcadia",
+        "link_text": null,
+        "link_url": null,
+        "avatar_url": null,
+        "bio": "I've been using Python since two and a half years ago and I really fell in love with it. I really hope that I can share my knowledge with others and also become more knowledgeable myself along the way." 
+    },
+    {
+        "github_username": "greatbahram",
+        "name": "Bahram Aghaei",
+        "link_text": "Github",
```

Travis CI is expecting value assigned to "link_text" to be spelled "GitHub" and not "Github".
```suggestion
        "link_text": "GitHub",
```

---

That's a little bit weird, I didn't know that.

Thanks